go.mod

```diff
@@ -78,8 +78,11 @@ require (
 	google.golang.org/grpc v1.26.0
 	gopkg.in/natefinch/lumberjack.v2 v2.0.0
 	honnef.co/go/tools v0.0.1-2020.1.6 // indirect
+	inet.af/tcpproxy v0.0.0-20200125044825-b6bb9b5b8252
```

Comment:
It's better to make `globalkilltest` an independent package so that the tidb package will not depend this..